README.md

```diff
@@ -1,4 +1,4 @@
-![hipe4ml](./logo.svg)
+å![hipe4ml](./logo.svg)
```

Comment:
Is the å needed for some reason or is it a typo?

Reply:
CIao @fcatalan92, it was indeed a typo, I fixed it. Thanks for spotting!